src/components/Login/Login.js

```diff
@@ -41,25 +42,27 @@ class Login extends Component {
     const { type } = this.state;
 
     return (
-      <div>
-        <button active={type === 'Login'} onClick={() => this.changeForm('Login')}>
-          Login
-        </button>
-        <button active={type === 'Sign up'} onClick={() => this.changeForm('Sign up')}>
-          Sign up
-        </button>
-        <form onSubmit={this.handleSubmit}>
-          <h3>
-            You'll need to login to continue
-          </h3>
-          <input spaced name="email" type="email" placeholder="email" onChange={this.handleChange} />
-          <input spaced name="password" type="password" placeholder="password" onChange={this.handleChange} />
-          <div>
-            <button type="submit">{type}</button>
-            <button small onClick={this.props.hideLogin}>Cancel</button>
-          </div>
-        </form>
-      </div>
+      <Wrapper className="container notification">
+        <Container>
+          <button className="button is-primary" active={type === 'Login'} onClick={() => this.changeForm('Login')}>
+            Login
+          </button>
+          <button className="button is-primary" active={type === 'Sign up'} onClick={() => this.changeForm('Sign up')}>
```

Comment:
you won't be able to write Sign up in two words because our route that we've setup is /signup. In order for this code to word the routes must match so you'll be able to hit our api endpoint